src/main/java/interfaces/Content.java

```diff
@@ -0,0 +1,13 @@
+package interfaces;
+
+import java.util.List;
+import java.util.Map;
+
+public interface Content {
+
+    Map<Integer, String>getQuestions();
```

Comment:
1)Formatting issue: some of the lines are shifted, after diamond braces we should have a space